src/router.tsx

```diff
@@ -63,7 +65,14 @@ const setRoutes = (isAuth: boolean) =>
       ]
     : [
         { path: '/', element: <Navigate to='/authorization' /> },
-        { path: '/authorization', element: <AuthPage /> },
+        {
+          path: '/authorization',
+          element: (
+            <GoogleReCaptchaProvider {...reCaptchaConfig}>
```

Comment:
You don't use any dynamic data here, could this provider be inside AuthPage? or create wrapper component